app/assets/stylesheets/_steps.scss

```diff
@@ -168,10 +169,16 @@ $dark-grey: #999999;
     margin: -10px -10px 0;
     padding: 10px;
   }
+}
```

Comment:
Can you fix this up to the commit were the closing brace is missing.

Reply:
This was added as we're taking out the `change-notes--description` from `change-notes`